cardano-testnet/test/Spec/Plutus/Direct/ScriptContextEqualityMint.hs

```diff
@@ -0,0 +1,268 @@
+{-# LANGUAGE DisambiguateRecordFields #-}
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Spec.Plutus.Direct.ScriptContextEqualityMint
+  ( hprop_plutus_script_context_mint_equality
+  ) where
+
+import           Prelude
+
+import           Cardano.Api
+
+import           Control.Monad
+import           Data.Monoid (Last (..))
+import           Data.String
+import           Hedgehog (Property, (===))
+import           System.Environment (getEnvironment)
+import           System.FilePath ((</>))
+
+import qualified Data.Aeson as J
+import qualified Data.Aeson.Types as Aeson
+import qualified Data.Map.Strict as Map
+import qualified Data.Text as T
+import qualified Hedgehog as H
+import qualified Hedgehog.Extras.Stock.IO.Network.Sprocket as IO
+import qualified Hedgehog.Extras.Test.Base as H
+import qualified Hedgehog.Extras.Test.Concurrent as H
+import qualified Hedgehog.Extras.Test.File as H
+import qualified Hedgehog.Extras.Test.Process as H
+import qualified System.Directory as IO
+import qualified Test.Base as H
+import           Test.Process (execCreateScriptContext, execCreateScriptContext')
+import qualified Test.Process as H
+import           Testnet.Cardano (defaultTestnetOptions, testnet)
+import qualified Testnet.Cardano as TC
+import qualified Testnet.Conf as H
+
+{- HLINT ignore "Redundant <&>" -}
+{- HLINT ignore "Redundant return" -}
+{- HLINT ignore "Use let" -}
+
+hprop_plutus_script_context_mint_equality :: Property
+hprop_plutus_script_context_mint_equality = H.integration . H.runFinallies . H.workspace "chairman" $ \tempAbsBasePath' -> do
```

Comment:
I'm thinking rename the shell script name to match the haskell code or the reverse.  Currently one says equality and the other says equivalence.